doc/TBG_macros.cfg

```diff
@@ -188,6 +188,30 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# Live in situ visualization using ISAAC
+# --isaac.period PERIOD
+#   ↳ Initial period in which a image shall be rendered
```

Comment:
that's pretty but why don't we stick with ascii characters to avoid problems with super-weird locals of some bash users or at least with the style that is already present in the file? :)

we can also update the full file afterwards for superior styling, but please let us not leave it like a rag rug.